autofit/non_linear/abstract_search.py

```diff
@@ -452,6 +452,7 @@ def fit(
         info=None,
         pickle_files=None,
         log_likelihood_cap=None,
+        bypass_nuclear : bool = False

```

Comment:
Should nuclear really be the default?

Reply:
Nuclear mode also requires that the enviroment variable "PYAUTOFIT_NUCLEAR_MODE" = 1.

This is a switch to not use it when its on, which was useful for some reason.

I have added a doc.